functions/funtions.py

```diff
@@ -8,17 +8,16 @@
 def take_data(
```

Comment:
```suggestion
def preprocess_data(
```
Should this be renamed to `preprocess_data`? Since it preprocess the data ...